typescript/sdk/src/token/adapters/SealevelTokenAdapter.ts

```diff
@@ -484,6 +502,34 @@ export abstract class SealevelHypTokenAdapter
       this.warpProgramPubKey,
     );
   }
+
+  async getMedianPriorityFee(): Promise<number | undefined> {
```

Comment:
nit: documenting this returns micro lamports would probably be useful for future readers